sunbeam-python/sunbeam/provider/local/deployment.py

```diff
@@ -71,11 +70,14 @@ class LocalDeployment(Deployment):
     def __init__(self, **data):
         super().__init__(**data)
         if self.juju_account is None:
-            self.juju_account = self._load_juju_account()
+            # self.juju_account = self._load_juju_account()
+            self.juju_account = None
```

Comment:
If juju account is not getting loaded here, we can remove the lines from L#72 - L#80

Reply:
This was an oversight, disabled for local testing, I've put it back